src/main/java/core/DataProvidersBoard.java

```diff
@@ -0,0 +1,76 @@
+package core;
+
+import org.testng.annotations.DataProvider;
+
+public class DataProvidersBoard {
```

Comment:
you always provide something with data providers, "provide" is redundant

Reply:
Fixed.